sympy/stats/crv.py

```diff
@@ -150,42 +150,38 @@ def __call__(self, *args):
 
 
 class SampleExternalContinuous:
-    """Class consisting of the methods that are used to sample values of random
-    variables from external libraries."""
+    """Base Class to sample values of Continuous random variables from
+    external libraries."""
 
+    def __new__(cls, dist, size, library):
+        _sample_class = _get_sample_class[library]
+        return _sample_class(dist, size)
```

Comment:
What if library isn't supported?

Reply:
It will raise an error in distribution.sample() in not supported